README.md

```diff
@@ -1,80 +1,78 @@
 # make_prg
 
-A tool to create a PRG for input to [Pandora][pandora] and [Gramtools][gramtools] from a
-Multiple Sequence Alignment.
+A tool to create and update PRGs for input to [Pandora][pandora] and [Gramtools][gramtools] from a set of 
+Multiple Sequence Alignments.
 
 [TOC]: #
 
 ## Table of Contents
+- [Dependencies](#dependencies)
 - [Install](#install)
-  - [Conda](#conda)
-  - [Local](#local)
-  - [Container](#container)
```

Comment:
Even though there is a binary available we should still include instructions for other installation ways. In some circumstances I would rather install in conda or a container rather than a binary...